lib/process-services-cloud/src/lib/process-services-cloud.module.ts

```diff
@@ -87,9 +87,12 @@ export class ProcessServicesCloudModule {
         };
     }
 
-    static forChild(): ModuleWithProviders<ProcessServicesCloudModule> {
+    static forChild(preferenceServiceInstance?: PreferenceCloudServiceInterface): ModuleWithProviders<ProcessServicesCloudModule> {
```

Comment:
This seems a place for a simple InjectionToken whenever you have to pass a service that is needed, and it is a little confusing adding/removing the providing

Reply:
Hi @VitoAlbano, sorry I forgot to mention it here, I reverted this changes in next PR (I should revert it in this PR here though)

https://github.com/Alfresco/alfresco-ng2-components/pull/7569/files#r842588171

Reply:
I mentioned it in this PR :)

https://github.com/Alfresco/alfresco-ng2-components/pull/7568#discussion_r842608834